Makefile

```diff
@@ -29,11 +29,16 @@ export
 export EVERY_COMPOSE_FILE=--file docker-compose.yml \
 	--file ./test/docker-compose.unit-tests-js.yml \
 
+# This is used to send docker traces to Jaeger. This is primarily useful for debugging build time performance
+ifdef WITH_TRACING
+buildx_builder_args := --builder=builder --load
```

Comment:
Doesn't `--load` output a tarball? 

https://github.com/moby/buildkit#docker-tarball

Reply:
🤔 Hrmm... maybe not; I don't see that it does anything different 🤔